docs/admin/service-admin-premium-multi-geo.md

```diff
@@ -127,8 +127,12 @@ Large-storage format semantic models shouldn't be moved from the region where th
 
 * Multi-Geo doesn't support [Metrics in Power BI](/power-bi/create-reports/service-goals-introduction).
 
+* Workspaces with non Power BI Fabric items can not be moved between regions.  You must delete all the non-Power BI Fabric items before moving a workspace to a different region.  Once the workspace has been moved, it can up to 30 minutes before a non-Power BI items can be created.
```

Comment:
```suggestion
* Workspaces with non-Power BI Fabric items can't be moved between regions.  You must delete all the non-Power BI Fabric items before moving a workspace to a different region.  Once the workspace has been moved, it can take up to 30 minutes before a non-Power BI items can be created.
```